src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/telemetry/ProbationBookingCancelledTelemetryEvent.kt

```diff
@@ -27,15 +28,22 @@ class ProbationBookingCancelledTelemetryEvent private constructor(
 
   companion object {
     fun user(booking: VideoBooking, user: User): ProbationBookingCancelledTelemetryEvent {
-      require(user is PrisonUser || (user is ExternalUser && user.isProbationUser)) {
-        "Cannot only create probation cancelled metric for prison or probation users."
+      require(user is PrisonUser || user is ServiceUser || (user is ExternalUser && user.isProbationUser)) {
+        "Can only create probation cancelled metric for service, prison or court users."
       }
 
       require(user.username == booking.amendedBy) {
         "Cannot create probation cancelled metric, user does not match the cancelled by user."
       }
 
-      return ProbationBookingCancelledTelemetryEvent(booking, if (user is PrisonUser) "prison" else "probation")
+      val cancelledBy = when (user) {
+        is PrisonUser -> "prison"
+        is ExternalUser -> "probation"
+        is ServiceUser -> "service"
```

Comment:
Not sure if "service" makes sense here (same applies to court).

Technically it would have been a user on NOMIS so perhaps should be "prison"?